source/js/components/petition/petition.jsx

```diff
@@ -318,18 +315,6 @@ export default class Petition extends React.Component {
     });
   }
 
-  /**
-   * Performs very simple validation for emails.
-   * @param {string} input the string that should be validated as an email address
-   * @returns {boolean} true if the input is a legal-enough email address, else false
-   */
-  validatesAsEmail(input) {
```

Comment:
This function has to be removed as the HTML itself does the checking of the E-mail.